### DISCUSSION TITLE
USB not detected in Android Subsystem for Win11

### DISCUSSION THREAD

**Discussion by SpinningPulsar** [fhdm-dev/scoppy#122] [Q&A]
I set up the android Subsystem for Linux and managed to install scoppy. However, the USB simply doesn't get connected and show up in the App. The Pico does blink and I can connect it to Windows. It just doesn't connect to the Scoppy app in Android Subsystem. 
Can anybody help me?
Thanks a lot! 
![WhatsApp Image 2023-10-04 at 16 57 27_82c42abd](https://github.com/fhdm-dev/scoppy/assets/17098604/b6d8a7e1-f848-4321-b12c-f7a19d754bc7)
![image](https://github.com/fhdm-dev/scoppy/assets/17098604/e3111a5b-1e80-4337-815e-0c814666bc23)

**Comment by fhdm-dev** [comment]
Hi,

The WSA doesn't currently support Android USB. See [https://developer.amazon.com/docs/appstore-on-windows/compatibility-guide.html#unsupported-hardware-features](https://developer.amazon.com/docs/appstore-on-windows/compatibility-guide.html#unsupported-hardware-features)

AFAIK the only way you could possibly get the app to work in the WSA is to use a Pico W and connect wirelessly. However, recent versions of the WSA don't seem to support mDNS so you'd have to give the Pico W a static IP address and then change the relevent connection settings in the Scoppy App. Note: I haven't tried this.
